evadb/third_party/databases/types.py

```diff
@@ -136,3 +154,31 @@ def execute_native_query(self, query_string: str) -> DBHandlerResponse:
             NotImplementedError: This method should be implemented in derived classes.
         """
         raise NotImplementedError()
+
+    def select(self, table_name: str) -> DBHandlerResponse:
+        """
+        Returns a generator that yields the data from the given table, or the data.
+        Args:
+            table_name (str): name of the table whose data is to be retrieved.
+        Returns:
+            DBHandlerResponse: An instance of DBHandlerResponse containing the data, data generator, or an error message. Data is in a pandas DataFrame.
+
+        Raises:
+            NotImplementedError: The default implementation of this method is for sqlalchemy-supported data source. The data source that does not support sqlalchemy should overwrite this function.
+        """
+        try:
+            uri = self.get_sqlalchmey_uri()
+            # Create a metadata object
+            engine = create_engine(uri)
+            metadata = MetaData()
+
+            Session = sessionmaker(bind=engine)
+            session = Session()
+            # Retrieve the SQLAlchemy table object for the existing table
+            table_to_read = Table(table_name, metadata, autoload_with=engine)
+            result = session.execute(table_to_read.select()).fetchall()
```

Comment:
Minor: Do we need to return a generator?

Reply:
generator does not work the sqlalmechy session now . Check #924 for the past investigation we have done.

Reply:
Drop a comment in the code!